.github/workflows/ci.yml

```diff
@@ -3,6 +3,8 @@ name: CI
 
 on:
   push:
+    branches-ignore:
```

Comment:
I feel like this should just care about `master` branch instead of ignoring branches

Reply:
In this case I was explicitly disabling the _normal_ CI process just for this test branch while I was experimenting with kicking off different triggers.  I wanted to explicitly disable the "noise" of the CI process for multiple commits and other trigger actions to come.

I'm not sure I agree with restricting CI to just `master` -- isn't the whole point of CI to continuously build and find errors in all commits?  So if I'm committing changes from different feature branches, I would expect the CI process would be giving me feedback on my changes there just like master.  Thoughts?

Reply:
this is for the `on push` trigger. I think you actually want the `on pull_request` trigger instead.